### ISSUE TITLE
Incorrect Address

### ISSUE THREAD

**Issue by super3** [Storj/driveshare-gui#74]
Just ran the ```allthethings``` branch and noticed a bug. I started everything from scratch and got this:

```
C:\Users\work\AppData\Roaming\DriveShare\dataserv-client\dataserv-client.exe --config_path=C:\Users\work\AppData\Roaming\DriveShare/drives/91b818ceb30f1297e07195b614f6ede254d33ddf --store_path=D:\ --max_size=10GB farm
INFO dataserv_client.messaging 43: Query: http://status.driveshare.org/api/address

INFO dataserv_client.messaging 43: Query: http://status.driveshare.org/api/register/18Prv6CxFWFyvwbYRPDJFybq4mBkcDJRGT/18Prv6CxFWFyvwbYRPDJFybq4mBkcDJRGT

INFO dataserv_client.api 88: Registered on server 'http://status.driveshare.org'.
```

Notice the register command has the ```18Prv6CxFWFyvwbYRPDJFybq4mBkcDJRGT``` address twice. This is incorrect. The endpoint url is ```/api/register/<bitcoin address>/<(optional)payout address>```. I put ```1P1eHxxfhsLM3ufQXDuYZKGdM3oBM5jrLh``` as the payout in the GUI, but seems it did not make it to dataserv-client.

**Comment by gordonwritescode** [comment]
Ack

**Comment by gordonwritescode** [comment]
This will be closed by #65

**Comment by littleskunk** [comment]
fixed with driveshare-gui develop (0.4.0)

INFO dataserv_client.messaging 43: Query: http://status.driveshare.org/api/register/1KYki1iw2ErP3UFL3JKZNpab2nhe9KqBY7/16zw6QGFbfmEyrAXDQKbVFhMceZcHobdtP

**Comment by super3** [comment]
@littleskunk All the issues fixed by the new version will be automatically fixed when this is merged Monday.
